src/main/java/it/pagopa/interop/probing/eservice/operations/mapping/dto/UpdateEserviceProbingStateDto.java

```diff
@@ -1,26 +1,27 @@
 package it.pagopa.interop.probing.eservice.operations.mapping.dto;
 
 import java.util.UUID;
-
 import javax.validation.constraints.NotNull;
-
 import com.fasterxml.jackson.annotation.JsonProperty;
-
 import lombok.Builder;
 import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
 
+@Getter
+@Setter
```

Comment:
`@Data` already include getter and setter

Reply:
Done